packages/auth-providers/dbAuth/api/src/DbAuthHandler.ts

```diff
@@ -907,11 +907,11 @@ export class DbAuthHandler<
     let user
 
     if (credentialId) {
-      user = await this.dbCredentialAccessor
-        .findFirst({
-          where: { [webAuthnOptions.credentialFields.id]: credentialId },
-        })
-        .user()
+      const credential = await this.dbCredentialAccessor.findUnique({
```

Comment:
Switching to `findUnique` over `findFirst` could potentially be breaking for someone that didn't have a unique, or some other equivalent, index on the database column. 

I'm happy to make this change in a minor/patch. Not having a unique constraint on your webauthn id field feels like an underlying issue in the users code. Our generator/setup tool adds an `@id` on the prisma model so would include an index as needed here.